base/libgit2/remote.jl

```diff
@@ -35,7 +35,13 @@ end
 function url(rmt::GitRemote)
     url_ptr = ccall((:git_remote_url, :libgit2), Cstring, (Ptr{Void}, ), rmt.ptr)
     url_ptr == C_NULL && return ""
-    return  unsafe_string(url_ptr)
+    return unsafe_string(url_ptr)
+end
+
+function name(rmt::GitRemote)
+    name_ptr = ccall((:git_remote_name, :libgit2), Cstring, (Ptr{Void}, ), rmt.ptr)
+    name_ptr == C_NULL && return ""
```

Comment:
Like [this](https://libgit2.github.com/libgit2/#HEAD/group/remote/git_remote_create_anonymous).